app/build.gradle

```diff
@@ -21,11 +24,23 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
If you use Kotlin only for tests, you should add this dependency only as `androidImplementation`.